config/locales/notices/en.yml

```diff
@@ -37,6 +37,6 @@ en:
   session_expired_html: Please %{link} the page to log in.
   session_expired_link: refresh
   session_timedout: >
-    We signed you out due to inactivity. This helps keep your account safe.
-    Please sign in again.
+    We signed you out. For your security, %{app} automatically ends your session
+    when you haven’t moved to a new page for %{minutes} minutes.
```

Comment:
How about "when you're idle for"? "moved to a new page" implies that refreshing the same page doesn't count as activity, but it does.